.gitmodules

```diff
@@ -4,12 +4,12 @@
   branch = dev/emc
 [submodule "ccpp/framework"]
   path = ccpp/framework
-  url = https://github.com/NCAR/ccpp-framework
-  branch = main
+  url = https://github.com/uturuncoglu/ccpp-framework
+  branch = feature/xgrid_dev_host
```

Comment:
```suggestion
  branch = ccpp_framework_stub_merged_20220202
```

Reply:
In addition to `.gitmodules`, the submodule pointer for ccpp-framework is not or no longer correct. It needs to point to hash 425cc4f68c24e9f71932560e689c1f50dd6fc7f8 (head of branch ccpp_framework_stub_merged_20220202 in my ccpp-framework fork).

Reply:
@climbfuji should be fixed now.

Reply:
@climbfuji done